CIP-0144/README.md

```diff
@@ -0,0 +1,356 @@
+---
+CIP: 144
+Title: Wallet Connector API
```

Comment:
```suggestion
Title: Extensible Wallet Connector Framework
```
I feel maybe we can make the title a bit more specific
This CIP is defining a new framework for wallet connectors, whilst also proposing a connector API

Reply:
yes, that's good :rocket: but I still think the shorter form (`Wallet Connector -`) should be used for its compliant extensions (e.g. https://github.com/cardano-foundation/CIPs/pull/986#pullrequestreview-2624359783).